ims/company/classification_and_control.md

```diff
@@ -0,0 +1,71 @@
+For information relating to laptops, please contact the People Team
+
+## Acceptable use of assets
+For more information on this, please consult our Acceptable Use Policy.
+
+## Classification of information
+All Made Tech information is classified into one of four levels of classification: 
+- Highly Confidential 
+- Confidential 
+- Internal
+- Public
+
+This section provides an over view on what these classifications mean and how information should be handled.
+ 
+### Highly confidential
+Highly Confidential documents are the most sensitive or business critical, and are restricted to specific pre-approved individuals within Made Tech.
+
+Information considered Highly Confidential includes:
+ - Personal medical information or Personal Health Information (PHI)
+ - Personally Identifiable Information (PII)
+ - Any information identified as Special Category Information under GDPR 
+ - Company financial account numbers, financial account security codes, access codes, or passwords
+ - Restricted communications and information from the Directors, Senior Managers and advisors such as information relating to acquisitions or sensitive contractual and partnership negotiations
+
+The majority of staff will have no access to information considered Highly Confidential. The default access to these assets is ‘Access Denied’.
+
+### Confidential
+Confidential documents and information are private to Made Tech. Access to this information is restricted to approved users who require them to perform their job role.
+
+Information considered confidential includes:
+ - Client related information (e.g. source code)
+ - Client owned information that we process on their behalf
+ - Third party contracts
+ - Company information such as organisational structures
+ - Internal Made Tech management information and systems (HR and financial systems, asset management)
+ - Company records
+ - Licensed software and license keys
+
+The default access to these assets is ‘Access Restricted’ and is only provided to approved users in order to perform their job role.
+
+### Internal
+Internal documents are private to Made Tech and are available to all users with a Made Tech login. This information can be considered "general access" and is used daily by Made Tech employees and contractors.
+
+Information considered "Internal" includes:
+ - General Company information stored on Google Drive and GitHub services
+ - Made Tech Gmail
+ - Small Improvements public feedback
+ - Line management information in Charlie HR
+
+### Public
+Public documents are meant to be in the public domain and have no access controls. This information will not harm the organisation if released into the public domain, such as marketing material and public accounts.
+ 
+Public information does not need to be labelled and has no handling or communication restrictions imposed.
+
+# Related Standards, Policies and Processes
+ - [Acceptable Use Policy](aup.md)
```

Comment:
I like that this is the full title. We should do this consistently. Equally we should use full name in filenames too.